resources/hiding_ci/install_ena.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+# Copyright 2025 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# # SPDX-License-Identifier: Apache-2.0
+
+# fail if we encounter an error, uninitialized variable or a pipe breaks
+set -eu -o pipefail
+
+AMZN_DRIVER_VERSION="2.13.3"
```

Comment:
when will we need to upgrade that and how will we know that it's time?

Reply:
As far as I could tell this version number we're using for the build didn't matter too much, and as we're installing into the newly built kernel so I don't think we'll get a conflict with this version.